OpenRA.Mods.TS/UtilityCommands/LegacySequenceImporter.cs

```diff
@@ -0,0 +1,197 @@
+			if (!string.IsNullOrEmpty(bibShape))
+			{
+				Console.WriteLine("\tbib: " + bibShape.ToLowerInvariant());
+				Console.WriteLine("\t\tLength: 1");
+				Console.WriteLine("\t\tZOffset: -1024");
+			}
+
+			Console.WriteLine();
+		}
+
+		void ImportInfantrySequences(IniSection section, string sequence)
+		{
+			Console.WriteLine(section.Name + ":");
+
+			var image = section.GetValue("Image", string.Empty);
+			if (!string.IsNullOrEmpty(image))
+			{
+				Console.WriteLine("\tDefaults: " + image.ToLowerInvariant());
+				Console.WriteLine("\t\tTick: 80");
+			}
+
+			var cameo = section.GetValue("Cameo", string.Empty);
+			if (!string.IsNullOrEmpty(cameo))
+			{
+				Console.WriteLine("\ticon: " + cameo.ToLowerInvariant());
+				Console.WriteLine("\t\tOffset: 0,0");
+				Console.WriteLine("\t\tUseTilesetExtension: false");
+				Console.WriteLine("\t\tUseTilesetCode: false");
+			}
+
+			if (file.Sections.Any(s => s.Name == sequence.ToLowerInvariant()))
+			{
+				var sequenceSection = file.GetSection(sequence);
+				var guard = sequenceSection.GetValue("Guard", string.Empty);
+				if (!string.IsNullOrEmpty(guard))
+				{
+					Console.WriteLine("\tstand:");
+					ConvertStartLengthFacings(guard);
+				}
+
+				var walk = sequenceSection.GetValue("Walk", string.Empty);
+				if (!string.IsNullOrEmpty(walk))
+				{
+					Console.WriteLine("\trun:");
+					ConvertStartLengthFacings(walk);
+				}
+
+				var prone = sequenceSection.GetValue("Prone", string.Empty);
+				if (!string.IsNullOrEmpty(prone))
+				{
+					Console.WriteLine("\tprone-stand:");
+					ConvertStartLengthFacings(prone);
+				}
+
+				var crawl = sequenceSection.GetValue("Crawl", string.Empty);
+				if (!string.IsNullOrEmpty(crawl))
+				{
+					Console.WriteLine("\tprone-run:");
+					ConvertStartLengthFacings(crawl);
+				}
+
+				var fireProne = sequenceSection.GetValue("FireProne", string.Empty);
+				if (!string.IsNullOrEmpty(fireProne))
+				{
+					Console.WriteLine("\tprone-shoot:");
+					ConvertStartLengthFacings(fireProne);
+				}
+
+				var fireUp = sequenceSection.GetValue("FireUp", string.Empty);
+				if (!string.IsNullOrEmpty(fireUp))
+				{
+					Console.WriteLine("\tshoot:");
+					ConvertStartLengthFacings(fireUp);
+				}
+
+				var idle1 = sequenceSection.GetValue("Idle1", string.Empty);
+				if (!string.IsNullOrEmpty(idle1))
+				{
+					Console.WriteLine("\tidle1:");
+					ConvertStartLengthFacings(idle1);
+				}
+
+				var idle2 = sequenceSection.GetValue("Idle2", string.Empty);
+				if (!string.IsNullOrEmpty(idle2))
+				{
+					Console.WriteLine("\tidle2:");
+					ConvertStartLengthFacings(idle2);
+				}
+			}
+
+			Console.WriteLine();
+		}
+
+		void ConvertStartLengthFacings(string input)
+		{
+			var splitting = input.Split(',');
+			if (splitting.Length >= 3)
+			{
+				Console.WriteLine("\t\tStart: " + splitting[0]);
+				Console.WriteLine("\t\tLength: " + splitting[1]);
+				Console.WriteLine("\t\tFacings: " + splitting[2]);
```

Comment:
`splitting.Length == 4 ? Console.WriteLine("\t\tFacings: 1") : Console.WriteLine("\t\tFacings: 8")` please.

If I get it right, `splitting[2] - splitting[1]` would be OpenRA `Stride:` but I never seen infantry which either used this logic either used the `Stride` tag in OpenRA.